Code/RDBoost/CMakeLists.txt

```diff
@@ -24,6 +23,7 @@ rdkit_headers(Wrap.h PySequenceHolder.h
               DEST RDBoost)
 
 if(RDK_BUILD_PYTHON_WRAPPERS)
+cmake_policy(SET CMP0079 NEW)
```

Comment:
This causes cmake 3.12.0-rc1 to fail on Mac because of the error:
```
CMake Error at Code/RDBoost/CMakeLists.txt:26 (cmake_policy):
  Policy "CMP0079" is not known to this version of CMake.
```

Apparently, it was introduced in CMake version 3.13: https://cmake.org/cmake/help/v3.13/policy/CMP0079.html

Reply:
*sigh* I thought that cmake would just ignore policies it didn't know. Thanks for catching that I was wrong.